axonserver/src/main/java/io/axoniq/axonserver/message/command/CommandRegistrationCache.java

```diff
@@ -76,15 +76,24 @@ public void remove(ClientIdentification client, String command) {
      * @param command the name of the command
      * @param commandHandler the handler of the command
      */
-    public void add( String command, CommandHandler commandHandler) {
+    public void add(String command, CommandHandler commandHandler) {
+        logger.trace("Add command {} to {}", command, commandHandler.client);
+        add(command, commandHandler, 100);
+    }
+
+    private void add(String command, CommandHandler commandHandler, int loadFactor) {
         logger.trace("Add command {} to {}", command, commandHandler.client);
         ClientIdentification clientIdentification = commandHandler.getClient();
-        ConsistentHash consistentHash = consistentHashPerContext.computeIfAbsent(clientIdentification.getContext(), c -> new ConsistentHash());
-        if( ! consistentHash.contains(clientIdentification.getClient())) {
-            consistentHashPerContext.put(clientIdentification.getContext(), consistentHash.with(clientIdentification.getClient(), 100, cmd -> provides(clientIdentification,cmd)));
+        ConsistentHash consistentHash = consistentHashPerContext.computeIfAbsent(clientIdentification.getContext(),
+                                                                                 c -> new ConsistentHash());
+        if (!consistentHash.contains(clientIdentification.getClient())) {
+            consistentHashPerContext.put(clientIdentification.getContext(),
+                                         consistentHash.with(clientIdentification.getClient(),
+                                                             loadFactor,
+                                                             cmd -> provides(clientIdentification, cmd)));
         }
         logger.trace("Consistent hash = {}", consistentHashPerContext.get(clientIdentification.getContext()));
-        registrationsPerClient.computeIfAbsent(clientIdentification, key ->new CopyOnWriteArraySet<>()).add(command);
+        registrationsPerClient.computeIfAbsent(clientIdentification, key -> new CopyOnWriteArraySet<>()).add(command);
```

Comment:
Save the load factor in registrationsPerClient as it will need to be distributed to other AxonServer nodes.

Reply:
Done